### ISSUE TITLE
Lil Question

### ISSUE THREAD

**Issue by cChimney** [Gucci637/Fentanyl#23]
When it is unpatched, would it work in a server?

**Comment by Gucci637** [comment]
> When it is unpatched, would it work in a server?

Yes! You could get like 15 hits in a few seconds if you posted a meme or something.

> [#20 (comment)](https://github.com/Gucci637/Fentanyl/issues/20#issuecomment-1074128654) Look

This is not a duplicate.

**Comment by cChimney** [comment]
> > When it is unpatched, would it work in a server?
> 
> Yes! You could get like 15 hits in a few seconds if you posted a meme or something.
> 
> > [#20 (comment)](https://github.com/Gucci637/Fentanyl/issues/20#issuecomment-1074128654) Look
> 
> This is not a duplicate.

What language is this made in?

**Comment by ProSlitqrt** [comment]
> > > When it is unpatched, would it work in a server?
> > 
> > 
> > Yes! You could get like 15 hits in a few seconds if you posted a meme or something.
> > > [#20 (comment)](https://github.com/Gucci637/Fentanyl/issues/20#issuecomment-1074128654) Look
> > 
> > 
> > This is not a duplicate.
> 
> What language is this made in?

Look in image when is gen

**Comment by Gucci637** [comment]
> > > When it is unpatched, would it work in a server?
> > 
> > 
> > Yes! You could get like 15 hits in a few seconds if you posted a meme or something.
> > > [#20 (comment)](https://github.com/Gucci637/Fentanyl/issues/20#issuecomment-1074128654) Look
> > 
> > 
> > This is not a duplicate.
> 
> What language is this made in?

The builder is made in Python but mostly consists of C++ wrappers so basically C++.
The payload is made in a weird JavaScript-C# makeshift as the discord client & backend are sort of C#

However, the payload is really weird it's practically obfuscated & beatified assembly if that makes any sense?

**Comment by Gucci637** [comment]
Y'all can join the discord now btw https://discord.gg/NrbJU8MdF3
